downstream/modules/platform/proc-creating-the-custom-execution-environment-definition.adoc

```diff
@@ -0,0 +1,161 @@
+  #- src: files/custom.repo
+  #  dest: configs
+
+additional_build_steps:
+  prepend_base:
+    # copy a custom pip.conf to override the location of the PyPI content
+    - ADD _build/configs/pip.conf /etc/pip.conf
+    # remove the default UBI repository definition
+    - RUN rm -f /etc/yum.repos.d/ubi.repo
+    # copy the hub CA certificate and update the trust store
+    - ADD _build/configs/hub-ca.crt /etc/pki/ca-trust/source/anchors
+    - RUN update-ca-trust
+    # if needed, uncomment to add a custom RPM repository configuration
+    #- ADD _build/configs/custom.repo /etc/yum.repos.d/custom.repo
+
+  prepend_galaxy:
+    - ADD _build/configs/ansible.cfg ~/.ansible.cfg
+
+...
+----
++
+
+. Create an `ansible.cfg` file under the `files/` subdirectory that points to your {PrivateHubName}, and contains the token used to authenticate to the hub and download collections.
++
+----
+$ cat files/ansible.cfg
+[galaxy]
+server_list = private_hub
+
+[galaxy_server.private_hub]
+url = https://private-hub.example.com/api/galaxy/
+----
++
+. Create a `pip.conf` file under the `files/` subdirectory which points to the internal PyPI mirror (a web server or something like Nexus):
++
+----
+$ cat files/pip.conf
+[global]
+index-url = https://<pypi_mirror_fqdn>/
+trusted-host = <pypi_mirror_fqdn>
+----
++
+
+* OPTIONAL: If a `bindep.txt` file is being used to add RPMs the custom {ExecEnvShort}, create a `custom.repo` file under the `files/` subdirectory that points to your disconnected Satellite or other location hosting the RPM repositories.  If this step is necessary, uncomment the steps in the example `execution-environment.yml` file that correspond with the `custom.repo` file.
++
+
+The following example is for the UBI repos. Other local repos can be added to this file as well. The URL path may need to change depending on where the mirror content is located on the web server. 
++
+----
+$ cat files/custom.repo
+[ubi-8-baseos]
+name = Red Hat Universal Base Image 8 (RPMs) - BaseOS
+baseurl = http://<ubi_mirror_fqdn>/repos/ubi-8-baseos
+enabled = 1
+gpgkey = file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+gpgcheck = 1
+
+[ubi-8-appstream]
+name = Red Hat Universal Base Image 8 (RPMs) - AppStream
+baseurl = http://<ubi_mirror_fqdn>/repos/ubi-8-appstream
+enabled = 1
+gpgkey = file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+gpgcheck = 1
+----
++
+. Add the CA certificate used to sign the private automation hub web server certificate. If the private automation hub uses self-signed certificates provided by the installer:
++
+.. Copy the file `/etc/pulp/certs/pulp_webserver.`crt` from your private automation hub and name it `hub-ca.crt`.
+
+.. Add the `hub-ca.crt` file to the `files/`` subdirectory.
++
+
+If the {PrivateHubName} uses user-provided certificates signed by a certificate authority:
+
+.. Make a copy of that CA certificate and name it `hub-ca.crt`.
+.. Add the `hub-ca.crt` file to the `files/`` subdirectory.
```

Comment:
these render as bullet items "c" and "d", but the list numbering should restart here so they should be "a" and "b"

Reply:
Do you mean you want the main numbering to continue on? That would make "If the {PrivateHubName} uses" step 7, and "Create your python requirements.txt" step 8. That would also bring the indented list letter into line with your ask.